specification/electronic-prescription-service-api.yaml

```diff
@@ -312,18 +314,74 @@ paths:
                 Bundle:
                   value:
                     $ref: components/examples/errors/prescription-order-update/bundle-error.json
+  /FHIR/R4/Task:
+    post:
+      operationId: return / withdraw
+      summary: Send either a Task-dispense-notification-amend or Task-dispense-proposal-return to EPS
+      description: |
+        ## Overview
+        This endpoint can be used to return a downloaded prescription to Spine, so that another dispensing system can download it. 
+        It can also be used to withdraw a dispensed prescription to Spine to notify Spine that something has gone wrong with a prescription that has been dispensed. 
 
+        For information on how to create these messages, see [DigitalMedicines-Task](https://simplifier.net/guide/DigitalMedicines/Task).
+
+        This endpoint is only available in the sandbox environment.
+      tags:
+        - dispensing
+      parameters:
+        - $ref: '#/components/parameters/BearerAuthorization'
+        - $ref: '#/components/parameters/RequestID'
+        - $ref: '#/components/parameters/CorrelationID'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+            examples:
+              return:
+                value:
+                  $ref: components/examples/secondary-care/community/acute/nominated-pharmacy/clinical-practitioner/1-Task-Request-Return-200_OK.json
+              withdraw:
+                value:
+                  $ref: components/examples/secondary-care/community/acute/nominated-pharmacy/clinical-practitioner/1-Task-Request-Withdraw-200_OK.json
+      responses:
+        '200':
+          description: Action completed successfully
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+              examples:
+                example:
+                  value:
+                    $ref: '#/components/responses/200OperationOutcome'
+        '4XX':
+          description: |
+            An error occurred as follows:
+
+            | HTTP status | Error code         | Description             |
+            | ----------- | -------------------| ----------------------- |
+            | 400         | BAD_REQUEST        | Invalid request bundle. |
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+              examples:
+                example:
+                  value:
+                    $ref: components/examples/errors/release/validation-error.json
   /FHIR/R4/Task/$release:
     post:
       operationId: release
       summary: Send a prescription release request to EPS
       description: |
         ## Overview
-        Use this endpoint to send a prescription release request to EPS.
+        Use this endpoint to send a [Task-release-message](https://simplifier.net/guide/DigitalMedicines/release-duplicate-2) to EPS.
```

Comment:
Could do with a sentence explaining what the endpoint is for, e.g. Use this endpoint to download prescriptions to be dispensed.